build-info-extractor/src/main/java/org/jfrog/build/extractor/clientConfiguration/client/artifactory/ArtifactoryManager.java

```diff
@@ -240,8 +240,8 @@ public ArtifactoryUploadResponse upload(DeployDetails details, String logPrefix)
         return upload(details, logPrefix, null);
     }
 
-    public ArtifactoryUploadResponse upload(DeployDetails details, String logPrefix, Integer minChecksumDeploySizeKb) throws IOException {
-        Upload uploadService = new Upload(details, logPrefix, minChecksumDeploySizeKb, log);
+    public ArtifactoryUploadResponse upload(DeployDetails details, String logPrefix, Integer MinChecksumDeploySizeKb) throws IOException {
```

Comment:
MinChecksumDeploySizeKb --> minChecksumDeploySizeKb
(In java, variable names always start with a lower-case).